packages/ui/src/components/va-tabs/types/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ComputedRef } from 'vue'
+
+export const TabsViewKey = Symbol('TabsView')
+export const TabKey = Symbol('Tab')
+export const TabsServiceKey = Symbol('TabsService')
+
+export type ClassComputed = {
```

Comment:
This looks redundant, same for ComputedStyle

Reply:
Removed redundant types.